ApeWorX/ape
# fix: `.return_value` bugs with tuples, structs, and length

### What I did

Always use call-tree approach since debug_tt approach for complex values seems bugged (for now) and super hard to do correctly as the rhymthe and reason disappears when using hardhat.

### How I did it

rely on calltree

### How to verify it

ugh, see my test, had to copy in a bunch of raw data because was hard to get into to be bad in a simple way.

### Checklist
